model/ftn/wmscrpmd.ftn

```diff
@@ -1065,31 +1103,46 @@
      &   GRID_CORNER_LON, GRID_CORNER_LAT, GRID_MASK,                   &
      &   GRID_DIMS, GRID_SIZE, GRID_CORNERS, GRID_RANK)
       END IF
-      CHECKSIGNS=0
+      CHECKSIGNS=1
```

Comment:
Is it a part of your development/debug or it should be always checked? if not, please turn it back to 0 or let me know if you want to keep it.